src/Kreta/TaskManager/Domain/Model/Organization/Organization.php

```diff
@@ -46,4 +55,34 @@ public function slug() : OrganizationSlug
     {
         return $this->slug;
     }
+
+    public function owners() : array
+    {
+        return $this->owners->toArray();
+    }
+
+    public function addOwner(Owner $owner)
```

Comment:
Furthermore, we need to add a method that brings the use case of if you are a participant you can promote to owner